### DISCUSSION TITLE
Serialization of 'Closure' is not allowed

### DISCUSSION THREAD

**Discussion by misaf** [spatie/laravel-sluggable#254] [Q&A]
Hello,
I'm using spatie/laravel-sluggable on Laravel ^9 but after new update I got error Serialization of 'Closure' is not allowed when I want using extra scope for using multi tenancy via below code:

/**
     * Get the options for generating the slug.
     */
    public function getSlugOptions(): SlugOptions
    {
        return SlugOptions::create()
            ->generateSlugsFrom('name')
            ->saveSlugsTo('slug')
            ->extraScope(fn (Builder $builder) => $builder->where('tenant_id', app('currentTenant')->id));
    }

**Comment by erashdan** [comment]
if you are trying to run job or event and you're construct model attach `SerializesModel` trait to event or job
```
use SerializesModels;
```
